<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QDialog" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255,255,255);
}

QPushButton:hover {
background-color:  #F3F3F3;
}

QTextEdit:hover{
background-color: #F3F3F3
}</string>
  </property>
  <widget class="QTextEdit" name="S_searchtext">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="S_searchBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="S_tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>401</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
border: 2px solid #F3F3F3;
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::TargetMoveAction</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>30</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>38</number>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>재료명</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="S_BackBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>S_BackBtn</sender>
   <signal>clicked()</signal>
   <receiver>SearchWindow</receiver>
   <slot>goMainWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goMainWindow()</slot>
 </slots>
</ui>
